vizro-core/examples/visual-vocabulary/pages/time.py

```diff
@@ -70,8 +70,8 @@
         vm.Graph(
             figure=categorical_column(
                 gapminder.query("country == 'Nigeria' and year > 1970"),
-                y="gdpPercap",
                 x="year",
+                y="gdpPercap",
```

Comment:
This reminds me of an initiative I had about a year ago to standardise the order of `x` and `y` arguments in all the examples in our repository. Every time I see `y` before `x`, it just hurts my eyes! 😅

Reply:
Haha yes, I generally agree 😀